src/sql/workbench/api/node/mainThreadConnectionManagement.ts

```diff
@@ -49,6 +49,11 @@ export class MainThreadConnectionManagement implements MainThreadConnectionManag
 		return Promise.resolve(this._connectionManagementService.getActiveConnectionCredentials(connectionId));
 	}
 
+	public $getUriForConnection(connectionId: string): Thenable<string> {
```

Comment:
I wonder if this function should be in Connection Management Service?  I need to do this look up in the main thread, so I'll need to duplicate this logic in my function otherwise I think.